src/test/java/cms/gov/madie/measure/resources/MeasureTransferControllerTest.java

```diff
@@ -416,4 +437,34 @@ public void verifyIfOrganizationNamesAreUpdated() {
     assertEquals(
         "Innovaccer Url", persistedMeasure.getMeasureMetaData().getDevelopers().get(2).getUrl());
   }
+
+  @Test
+  public void testReorderGroupPopulations() {
+    Group copiedGroup = Group.builder().populations(groups.get(0).getPopulations()).build();
+
+    List<Group> reorderedGroups = controller.reorderGroupPopulations(groups);
+
+    assertEquals(1, reorderedGroups.size());
+    assertEquals(4, copiedGroup.getPopulations().size());
+    assertEquals(6, reorderedGroups.get(0).getPopulations().size());
```

Comment:
If the original group only has 4 populations, then why are adding 2 new populations ?